src/engine/sparqlExpressions/SparqlExpressionValueGetters.h

```diff
@@ -140,6 +140,20 @@ struct StringValueGetter : Mixin<StringValueGetter> {
     return std::string(asStringViewUnsafe(s.getContent()));
   }
 };
+// Similar to `StringValueGetter`, but correctly escapes and unescapes string
+// sequences.
```

Comment:
You have to be a bit more precise here, explain exactly (with an example) what it does and why it is necessary.